src/Core/src/Handlers/View/ViewHandler.cs

```diff
@@ -15,35 +15,113 @@
 
 namespace Microsoft.Maui.Handlers
 {
-	public partial class ViewHandler
+	public abstract partial class ViewHandler : IViewHandler
 	{
 		public static PropertyMapper<IView> ViewMapper = new PropertyMapper<IView>
 		{
 			[nameof(IView.BackgroundColor)] = MapBackgroundColor,
 			[nameof(IView.Frame)] = MapFrame,
 			[nameof(IView.IsEnabled)] = MapIsEnabled,
-			[nameof(IView.AutomationId)] = MapAutomationId
+			[nameof(IView.AutomationId)] = MapAutomationId,
+			[nameof(IView.Semantics)] = MapSemantics
 		};
 
+		internal ViewHandler() {}
+
+		bool _hasContainer;
+
+		public bool HasContainer
+		{
+			get => _hasContainer;
+			set
+			{
+				if (_hasContainer == value)
+					return;
+
+				_hasContainer = value;
+
+				if (value)
+					SetupContainer();
+				else
+					RemoveContainer();
+			}
+		}
+
+		protected abstract void SetupContainer();
+
+		protected abstract void RemoveContainer();
+
+		public IMauiContext? MauiContext { get; private set; }
+
+		public object? NativeView
+		{
+			get;
+			private protected set;
+		}
+
+		public IView? VirtualView 
+		{ 
+			get; 
+			private protected set; 
+		}
+
+		public void SetMauiContext(IMauiContext mauiContext) => MauiContext = mauiContext;
+
+		public abstract void SetVirtualView(IView view);
+
+		public abstract void UpdateValue(string property);
+
+		void IViewHandler.DisconnectHandler() => DisconnectHandler(((NativeView?)NativeView));
+
+		public abstract Size GetDesiredSize(double widthConstraint, double heightConstraint);
+
+		public abstract void SetFrame(Rectangle frame);
+
+		private protected void ConnectHandler(NativeView? nativeView)
+		{
+		}
+
+		private protected void DisconnectHandler(NativeView? nativeView)
```

Comment:
Why is this private? Is there some reason for this not to flow?

Reply:
My thinking is not to expose too many versions of this out to 3rd party libraries. I made the constructor on ViewHandler internal so that people can't inherit from ViewHandler they have to inherit from the generic ones. 

I don't think there's a point for users to inherit from the non generic ViewHandler 

Because of this I didn't want 3rd party libraries to inherit from ViewHandler<T,U> and see two different ConnectHandler and DisconnectHandler calls. I was thinking it was best if they only interacted with the Typed one that's part of ViewHandler<T,U>